vite.config.js

```diff
@@ -0,0 +1,92 @@
+import { defineConfig, splitVendorChunkPlugin } from 'vite'
+import vue from '@vitejs/plugin-vue2'
+import legacy from '@vitejs/plugin-legacy'
+import { createSvgIconsPlugin } from 'vite-plugin-svg-icons'
+import path from 'node:path'
+
+// / <reference types="vitest" />
+// / <reference types="vite/client" />
+export default defineConfig({
+  css: {
+    preprocessorOptions: {
+      scss: {
+        additionalData: `
+          @import "@/assets/scss/custom.scss";
+        `
+      }
+    }
+  },
+
+  plugins: [
+    vue({
+      target: 'modern',
+      vueTemplateOptions: {
+        compilerOptions: {
+          whiteSpace: 'condense'
+        },
+
+        transformAssetUrls: {
+          video: ['src', 'poster'],
+          source: ['src'],
+          img: ['src'],
+          image: ['xlink:href', 'href'],
+          use: ['xlink:href', 'href'],
+          'b-avatar': 'src',
+          'b-img': 'src',
+          'b-img-lazy': 'src',
+          'b-card': 'img-src',
+          'b-card-img': 'src',
+          'b-card-img-lazy': ['src', 'blank-src'],
+          'b-carousel-slide': 'img-src',
+          'b-embed': 'src'
+        }
+      }
+    }),
+
+    legacy({
+      targets: [
+        'defaults and not IE 11'
```

Comment:
85.2% of the audience coverage in TW.
See [browserl.ist](https://browsersl.ist/#q=defaults+and+not+IE+11&region=TW)